k6ext/panic.go

```diff
@@ -36,3 +44,20 @@ func Panic(ctx context.Context, format string, a ...interface{}) {
 	_ = p.Release()
 	_ = p.Kill()
 }
+
+type userFriendlyError struct{ err error }
+
+func (e *userFriendlyError) Unwrap() error { return e.err }
+
+func (e *userFriendlyError) Error() string {
+	switch {
+	default:
+		return e.err.Error()
+	case e.err == nil:
+		return ""
+	case errors.Is(e.err, context.DeadlineExceeded):
+		return strings.ReplaceAll(e.err.Error(), context.DeadlineExceeded.Error(), "timed out")
```

Comment:
Sorry, I didn't get what you meant. This code only converts a `context deadline exceeded` with `timeout` if the context is canceled because of a timeout to print a user-friendly message.

Reply:
🤦 I was getting confused with `ContextClosed`. LGTM 👍